src/Microsoft.Framework.NullLogging.Internal/NullLogger.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Framework.Logging;
+
+namespace Microsoft.Framework.Internal
+{
+    public class NullLogger : ILogger
```

Comment:
https://github.com/aspnet/Common/pull/2#discussion_r25474847 the common repo has all shared sources